tests/test_vicarimage.py

```diff
@@ -141,18 +143,18 @@ def test_VicarImage(self):
 
         # binheader
         test = VicarImage(test_dir / 'C2069302_GEOMA.DAT')
-        self.assertIsNone(test.array)
-        self.assertIsNone(test.array2d)
-        self.assertIsNone(test.array3d)
-        self.assertIsNone(test.prefix)
-        self.assertIsNone(test.prefix2d)
-        self.assertIsNone(test.prefix3d)
+        self.assertTrue(test.array is None)
```

Comment:
Why do you not like `self.assertIsNone`?

Reply:
Changed.